### PR TITLE
RATIS-2169. mvnw test: Corrupted STDOUT by directly writing to native stream in forked JVM 1

### PR DESCRIPTION
**Pull request by szetszwo**
See RATIS-2169

### CODE REVIEW

**Comment by szetszwo** [comment]
The failed test is a known problem but not related this; see RATIS-2151.

File: pom.xml [hunk 1]
```diff
@@ -176,7 +176,7 @@
     <maven-checkstyle-plugin.version>3.3.0</maven-checkstyle-plugin.version>
     <maven-clover2-plugin.version>4.0.6</maven-clover2-plugin.version>
     <maven-pdf-plugin.version>1.6.1</maven-pdf-plugin.version>
-    <maven-surefire-plugin.version>3.0.0-M4</maven-surefire-plugin.version>
+    <maven-surefire-plugin.version>3.5.1</maven-surefire-plugin.version>
```
Comment (adoroszlai) [inline pom.xml:179]: Versions newer than 3.0.0-M4 are affected by https://issues.apache.org/jira/browse/SUREFIRE-1722

Maybe we have to decide which bug to live with ;)

File: pom.xml [hunk 2]
```diff
@@ -223,8 +223,8 @@
     <testsThreadCount>4</testsThreadCount>
 
     <slf4j.version>2.0.7</slf4j.version>
-    <junit.jupiter.version>5.10.1</junit.jupiter.version>
-    <jacoco.version>0.8.11</jacoco.version>
+    <junit-bom.version>5.11.2</junit-bom.version>
+    <jacoco.version>0.8.12</jacoco.version>
```
Comment (adoroszlai) [inline pom.xml:227]: According to the comments on stack overflow:

> upgrading jacoco-plugin from 0.8.1 to 0.8.4 did the job

But we already have 0.8.11, which is newer than both, so I guess this won't fix the problem.

We can upgrade anyway, Ozone has been using 0.8.12 for 6 months now (HDDS-10660).

**Review by adoroszlai — COMMENTED** [review]
Thanks @szetszwo for the patch.

Did it happen only locally or in some PR?  I don't see such issue in [logs collected](https://github.com/adoroszlai/ratis-build-results) from runs on `master`.

File: pom.xml (hunk 1 above)
Comment (szetszwo) [reply pom.xml:179]: Good to know.  Let's remain in 3.0.0-M4 then.

File: pom.xml (hunk 2 above)
Comment (szetszwo) [reply pom.xml:227]: I am also aware that.  Just wonder they might have a better fix in a newer version.

**Review by adoroszlai — APPROVED** [review]

**Pull request merged**